docs/client/concepts/parameter-stores.mdx

```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
 
-## How should I use Parameter Stores?
-Here is a suggested workflow:
-1. Create a parameter store for you team or project.  Parameter Stores are an object to hold related parameters.
-2. As you are adding to your mobile app, consider which variables you'd like to be controlled on the server-side, rather than hard coded into your app. These can be things like:
+---
+
+### **What Exactly is a Parameter Store?**
+
+Parameter Stores let you manage the parameters driving features in your app (boolean flags, text resources, numerical settings) without hardcoding them into your code. This allows you to dynamically adjust app behavior by decoupling configuration from code.
+
+Unlike Feature Gates, Experiments, or Dynamic Configs tied to specific workflows, Parameter Stores offer the flexibility to define and manage parameters independently, which can be remapped to these features later.
+
+---
+
+### **How to Use Parameter Stores: A Practical Example**
+
+Let’s say you want to implement a new onboarding flow, but only for users on the latest version of your app. Here’s how Parameter Stores help without needing constant code updates:
+
+1. **Create a Boolean Parameter**
+
+   In the Statsig Console, create a boolean parameter called `enable_new_onboarding` and set its default value to `false`. This will be your feature toggle.
+
+2. **Use the Parameter in Your Code**
+
+   Fetch the value of the parameter in your app using the Statsig SDK:
+
+   ```java
+   boolean newOnboardingEnabled = StatsigClient.getConfig("enable_new_onboarding").getBoolean("value", false);
+   if (newOnboardingEnabled) showNewOnboardingFlow();
+   else showOldOnboardingFlow();
+   ```
+
+   The app reacts dynamically based on whether the new onboarding flow is enabled, without hardcoding the condition.
+
+3. **Map the Parameter to a Feature Gate**
+
+   Once you're ready to roll out the new onboarding flow to a specific user segment, remap the `enable_new_onboarding` parameter to a Feature Gate that targets users on the latest version.
+
+4. **Control Updates in Real-Time**
 
-- A boolean parameter to control access to a new feature.  Even though you may be used to using Feature Gates for boolean based feature management, start with a boolean parameter instead
-- A string parameter for a text resource that you may want to swap or experiment with
-- A number parameter that you use as input to an onboarding flow (onboarding_steps), a list length to truncate at, etc
-... and many more
+   When you're ready to release the new flow to more users, simply update the parameter in the Statsig Console—no need to release a new app version.
```

Comment:
This example still doesnt sell it- you could just use a feature gate in this example